bin/get-task-links

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+## This script get all tasks for SAPS interacting with Dispatcher backend.
```

Comment:
Is this doc correct? the script is named `get-task-links` the doc says nothing about links

Reply:
Fixed